paasta_tools/marathon_serviceinit.py

```diff
@@ -39,6 +39,17 @@
 log.addHandler(logging.NullHandler())
 
 
+class MarathonDeployStatus:
+    """ An enum to represent marathon app deploy status """
+    Running, Deploying, Stopped, Delayed, Waiting = range(0, 5)
+
+    @classmethod
+    def tostring(cls, val):
+        for k, v in vars(cls).iteritems():
```

Comment:
Might be worth a comment that the name of the keys (Running, Deployed, etc) can't be changed without affecting the API with this approach